app/src/visionglass/res/layout/visionglass_layout.xml

```diff
@@ -106,10 +106,27 @@
                 app:iconGravity="textStart"
                 app:iconPadding="0dp"
                 app:iconTint="@color/white"
-                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintEnd_toStartOf="@id/window_distance_button"
                 app:layout_constraintStart_toEndOf="@id/realign_button"
                 app:layout_constraintTop_toTopOf="parent" />
 
+            <com.google.android.material.button.MaterialButton
+                android:id="@+id/window_distance_button"
+                android:layout_width="wrap_content"
+                android:layout_height="?attr/actionBarSize"
+                android:backgroundTint="@color/azure"
```

Comment:
If you want to set the button as checked/unchecked, perhaps use this instead:

```xml
android:backgroundTint="@color/bg_button_checkable"
```

However, the problem with this background is that the button appears blue when checked and grey when unchecked. This is how the headlock button works but maybe we want things to look different in this case.

Reply:
This is an action button, so I don't think we want to have a checked/unchecked state.